docs/content/users/extend/custom-commands.md

```diff
@@ -218,11 +218,11 @@ If your container command should run from the directory you are running the comm
 
 Example: `## HostWorkingDir: true`
 
-### “ExecRaw” Annotation (Container Commands Only)
+### `ExecRaw` Annotation (Container Commands Only)
```

Comment:
@rfay This fixes the spell check issue to the detriment of the reader by not following the format of the page. I’d either wrap all the heading annotations in backticks (`“HostWorkingDir” Annotation` → <code>`HostWorkingDir` Annotation</code>, or add “ExecRaw” to the exclusion list.

Reply:
I don't think `ExecRaw` should be in the spellcheck list because it's not a word.

Reply:
These aren’t words either:

- `ProjectTypes`
- `OSTypes`
- `HostBinaryExists`
- `DBTypes`
- `HostWorkingDir`

But they’re quoted in headings on this page, just as `ExecRaw` was. I’m saying that if you’re changing the heading format to appease the spell checker, you might as well update the format for the rest of them so you’re not signaling to the reader that `ExecRaw` is somehow different. (Unless this is intentional and meaningful break in format and I’m missing the point.)

Reply:
Hmm, so I probably added all those to spellcheck huh? In that case I'll just do that.

Reply:
Sure, or use backticks for the rest of the headings.

> I’d either wrap all the heading annotations in backticks (`“HostWorkingDir” Annotation` → <code>`HostWorkingDir` Annotation</code>, etc.), or add “ExecRaw” to the exclusion list.

I’m not sure how else to say this; whatever the answer, I think the reader that should matter more than the spell checker. The consistency of the page is for the reader.

Reply:
Fixed in https://github.com/ddev/ddev/pull/4746, you don't need to say it any more :)